src/inc/clrnt.h

```diff
@@ -835,7 +835,7 @@ RtlVirtualUnwind_Unsafe(
 //  X86
 //
 
-#if defined(_TARGET_X86_)
+#if defined(_TARGET_X86_) && defined(WIN32)
```

Comment:
Sounds good to me.

> it seems the warning disabling pragmas are outdated, there is no function definition between the pragmas.

In this case, @seanshpark can you please remove the pragmas as well?